.gitattributes

```diff
@@ -27,26 +27,25 @@ pkg/fleet/internal/fixtures/** text=auto eol=lf
 #	textconv = astextplain
 # ```
 
-
-*.doc	diff
-*.DOC	diff
-*.docx	diff
-*.DOCX	diff
-*.docm	diff
-*.DOCM	diff
-*.dot	diff
-*.DOT	diff
-*.dotx	diff
-*.DOTX	diff
-*.dotm	diff
-*.DOTM	diff
-*.pdf	diff
-*.PDF	diff
-*.rtf	diff
-*.RTF	diff
-*.ods	diff
-*.ODS	diff
-*.odf	diff
-*.ODF	diff
-*.odt	diff
-*.ODT	diff
+*.doc diff
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Is this format change expected?